File: PHP/Items.php
<?php
class Items {
    private $id;
    private $nama_items;
    private $harga;
    private $foto;

    public function __construct($id = "", $nama = "", $harga = 0, $foto = "uploads/default.jpg") {
        $this->id = $id;
        $this->nama_items = $nama;
        $this->harga = $harga;
        $this->foto = $foto;
    }

    // Setter
    public function setId($id) { $this->id = $id; }
    public function setNamaItems($nama_items) { $this->nama_items = $nama_items; }
    public function setHarga($harga) { $this->harga = $harga; }
    public function setFoto($foto) { $this->foto = $foto; }

    // Getter
    public function getId() { return $this->id; }
    public function getNamaItems() { return $this->nama_items; }
    public function getHarga() { return $this->harga; }
    public function getFoto() { return $this->foto; }
}
?>

File: Python/Items.py
class Items:
    def __init__(self, id="", nama_items="", harga=0):
        self.__id = id
        self.__nama_items = nama_items
        self.__harga = harga

    # Setter
    def set_id(self, id):
        self.__id = id

    def set_nama_items(self, nama_items):
        self.__nama_items = nama_items

    def set_harga(self, harga):
        self.__harga = harga

    # Getter
    def get_id(self):
        return self.__id

    def get_nama_items(self):
        return self.__nama_items

    def get_harga(self):
        return self.__harga

    # Tambahan method sesuai C++
    def id_length(self):
        return len(self.__id)

    def name_length(self):
        return len(self.__nama_items)

    def harga_length(self):
        return len(str(self.__harga))

    # Destructor opsional
    def __del__(self):
        pass